scripts/gen_checkerboard.py

```diff
@@ -261,6 +289,8 @@ def main(args):
 parser.add_argument("--dataroot", help="Root location of the input data", type=str, required=True)
 parser.add_argument("--dataout", help="Root location of the output data", type=str, required=True)
 parser.add_argument("--box_size", help="Size of the bounding box", type=int, required=False, default=100)
+parser.add_argument("--based_on", help="This determines the shape of synthetic data array", type=str, required=False, default='dutch_f3')
+parser.add_argument("--box_number", help="Number of boxes", type=int, required=False, default=2)
```

Comment:
you can add a "choices" parameter like in "--type" below and drop the assert in the code